tracee-ebpf/tracee/internal/bufferdecoder/protocol.go

```diff
@@ -88,7 +88,7 @@ func (MprotectWriteMeta) GetSizeBytes() uint32 {
 
 // SlimCred struct is a slim version of the kernel's cred struct
 // it is used to unmarshal binary data and therefore should match (bit by bit) to the `slim_cred_t` struct in the ebpf code.
-// ANY CHANGE TO THIS STRUCT WILL BE REQUIRED ALSO TO external.SlimCred
+// ANY CHANGE TO THIS STRUCT WILL BE REQUIRED ALSO TO types.SlimCred
```

Comment:
`// ANY CHANGE TO THIS STRUCT WILL BE REQUIRED ALSO TO types.SlimCred and bufferdecoder.SlimCred`